content/docs/start/experiments.md

```diff
@@ -120,6 +120,33 @@ Experiment results have been applied to your workspace.
 
 <details>
 
+### ℹ️ More information about (Hyper)parameters
+
+It's pretty common for data science projects to include configuration files that
+define adjustable parameters to train a model, adjust model architecture, do
+pre-processing, etc. DVC provides a mechanism for stages to depend on the
+specific variables from a file.
+
+By default, DVC assumes a parameters file named `params.yaml` is available in
+your project. DVC parses this file and creates dependencies to the variables
+(`model.conv_units` and `train.epochs`) found in the file.
+
+Here is the contents of `params.yaml` file:
```

Comment:
💅 could skip this sentence/paragraph by just adding "Example:" in the previous paragraph.

Reply:
Done so in 800a11cfe